src/main/java/odd/jobs/entities/user/User.java

```diff
@@ -64,4 +64,12 @@ public boolean isCredentialsNonExpired() {
     public boolean isEnabled() {
         return !isBlocked;
     }
+
+    public String getFirstName() {
+        return firstName;
+    }
+
+    public String getLastName() {
+        return lastName;
+    }
```

Comment:
Przeciez mamy lomboka, wywal getery

Reply:
Zauwaz, ze getery w niektorych miejscach musza byc, bo ta klasa implementuje taki specjalny interfejs, ktory mowi Springowi ze to jest nasz User